pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -132,11 +132,21 @@ type Reconciler struct {
 	cloudEventClient  cloudevent.CEClient
 	metrics           *pipelinerunmetrics.Recorder
 	pvcHandler        volumeclaim.PvcHandler
+
+	// disableResolution is a flag to the reconciler that it should
+	// not be performing resolution of pipelineRefs.
+	// TODO(sbwsg): Once we've agreed on a way forward for TEP-0060
+	// this can be removed in favor of whatever that chosen solution
+	// is.
+	disableResolution bool
 }
 
 var (
 	// Check that our Reconciler implements pipelinerunreconciler.Interface
 	_ pipelinerunreconciler.Interface = (*Reconciler)(nil)
+
+	// Indicates pipelinerun resolution hasn't occurred yet.
+	errResourceNotResolved = fmt.Errorf("pipeline ref has not been resolved")
```

Comment:
nit: `errors.New` when there's no formatting.

🤔  I thought this was something golangci-lint would complain about, sorry I didn't notice before.

Also cc @n3wscott in case there's something useful in the controller framework we can use here to simplify any of the logic.  None of the things that come to mind for me seem quite right.